test/.eslintrc

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
That's right. The only practical way to configure e.g. [linter-eslint](https://atom.io/packages/linter-eslint#using-eslint) (Atom) or [SublimeLinter](https://github.com/roadhump/SublimeLinter-eslint#settings) is with a local `.eslintrc` file in the project.